.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: GitHub Actions CI
+
+on: [push, pull_request]
+
```

Comment:
```suggestion
env:
  CI: true

```

Reply:
Isn't this set automatically?

Reply:
Nope.

Reply:
Is this environment variable explicitly needed for this CI run? I generally try to avoid setting unnecessary environment variables, and it wasn't defined in the existing `.travis.yml`.

Reply:
It certainly does not hurt and it's what you had on Travis CI by default before.

Reply:
And in most if not all CI providers :P

Reply:
If collaborators would like this added, I'd be happy to have it committed. Still cautious about adding additional environment variables that may be unnecessary.

Reply:
Well, it's not unnecessary. Many modules use this var to skip installing (like precommit packages) or showing messages on postinstall. And you had it already there (maybe without knowing).

Reply:
Aye. `airtap` also uses it.

Reply:
Let’s add it then!

Reply:
Not (or no longer) needed: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables